tests/test_definition.py

```diff
@@ -51,12 +51,6 @@ def test_definition_from_general_config(workflow_folder):
             assert "Region A" in obs.region
         # imported from https://github.com/IAMconsortium/common-definitions repo
         assert "World" in obs.region
-        # added via general-config definitions
-        assert "Austria" in obs.region
```

Comment:
If you introduce the pin for common-definitions, you can bring this one back (I think)